src/main/java/com/aca/armine/serialization/Employee.java

```diff
@@ -1,5 +1,97 @@
 package com.aca.armine.serialization;
 
+import java.math.BigDecimal;
+import java.time.LocalDateTime;
+
 public class Employee {
+    private String id;
+    private Integer index;
+    private BigDecimal balance;
+    private BigDecimal salary;
+    private Integer age;
+    private String email;
+    private String phone;
+    private LocalDateTime birthDate;
+    private String section;
+    private BigDecimal newSalary= BigDecimal.ZERO;
```

Comment:
why this property?

Reply:
and you have additional redundant parts, for instance in the increasing salaries